ciao-controller/compute.go

```diff
@@ -452,6 +452,21 @@ func instanceToServer(context *controller, instance *types.Instance) (payloads.S
 	return server, nil
 }
 
+// returnErrorCode returns error codes for the http call
+func returnErrorCode(w http.ResponseWriter, httpError int, message string) {
+	var returnCode payloads.HTTPReturnErrorCode
+	returnCode.Error.Code = httpError
+	returnCode.Error.Name = http.StatusText(returnCode.Error.Code)
+	returnCode.Error.Message = message
+
+	b, err := json.Marshal(returnCode)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
Shouldn't there be a return here?  Otherwise we'll send two errors.  Not quite sure what would happen.